include/seastar/net/api.hh

```diff
@@ -169,6 +169,12 @@ struct session_dn {
     sstring issuer;
 };
 
+  /// Information about a certificate
```

Comment:
Please add some motivation to the changelog.

Reply:
In the commit message, yeah? Or is there a separate changelog somewhere?

Reply:
The commit message.